### DISCUSSION TITLE
Retrieving timestamp and Frame count data using spinnaker via bonsai

### DISCUSSION THREAD

**Discussion by bruno-f-cruz** [bonsai-rx/bonsai#976] [Q&A]
The following question was asked in the Bonsai Users - Google Groups
[Link to original post](https://groups.google.com/g/bonsai-users/c/k6p9BW9yAe4)

---
**Original post**

Hi all

I have been using Bonsai+FlyCapture before, but I would like to switch to SpinnakerCapture given that new pointgrey cameras only support Spinnaker. Unfortunately, Spinnaker does not support embedding timestamp and counter information like FlyCapture, but has a chunk data embedded in the frame. I was wondering how one could retrieve that chunk data, and ultimately the timestamp and frame counter data via Bonsai. Any help would be appreciated!!

Jay

---

**Comment by bruno-f-cruz** [comment]
Hey, you can access it in the output of `SpinnakerCapture` by right clicking the operator. The fields you are looking for are probably the following:

![image](https://user-images.githubusercontent.com/7049351/179931561-0e36e409-3166-4521-b21c-ba26fb173736.png)

Hope it helps

**Reply by jel0624** [reply]
thanks!! this is exactly what I needed
